constitution.md

```diff
@@ -78,10 +80,9 @@ The **President** shall:
 
 The **Vice President** shall:
 1. Report to the President
```

Comment:
Some( but not all) of these `Report to <name>` sentences dont have `.` at the end but all other lines do. Not sure what to do, just wanna be consistent